release-tools/prow.sh

```diff
@@ -1064,15 +1064,21 @@ main () {
                 docker tag "$i:latest" "$i:csiprow" || die "tagging the locally built container image for $i failed"
             done
 
-            if [ -e deploy/kubernetes/rbac.yaml ]; then
+            if [ -e "$(pwd)/deploy/kubernetes/rbac.yaml" ] || [ -e "$(pwd)/deploy/kubernetes/csi-snapshotter/rbac-csi-snapshotter.yaml" ]; then
```

Comment:
this check: `if [ -e deploy/kubernetes/rbac.yaml ];` does not work without $(pwd), so this code path wasn't being executed. Also, we needed to adjust for the snapshot-controller by removing it from `cmds`